docs/diagrams/StorageClassDiagram.puml

```diff
@@ -4,7 +4,7 @@ skinparam arrowThickness 1.1
 skinparam arrowColor STORAGE_COLOR
 skinparam classBackgroundColor STORAGE_COLOR
 
-package Storage{
+package Storage as StoragePackage{
```

Comment:
How does this affect?

Reply:
The code blow has a class named Storage. Without using the 'as ...', the other arrows that which are meant to refer to the 'Storage Class' ends up referring to this 'Storage Package'. Leading to the situation below.

![image](https://user-images.githubusercontent.com/39543273/230615606-052918f1-6fc2-4c58-866d-82a7229492a9.png)